ondewo/t2s/text-to-speech.proto

```diff
@@ -205,8 +207,75 @@ message RequestConfig {
         // The default value is the language of the pipeline.
         string normalizer = 8;
     }
+
+    // t2s_service_config provides the configuration of the service such as API key, bearer tokens, JWT,
+    // and other header information as key value pairs, e.g., <pre><code>MY_API_KEY='LKJDIFe244LKJOI'</code></pre>
+    optional google.protobuf.Struct t2s_service_config = 9;
+
+    oneof oneof_t2s_cloud_provider_config {
```

Comment:
Shouldn't this be different such as

`message RequestConfig {
...
T2sCloudProviderConfig t2s_cloud_provider_config = 10;
}

message T2sCloudProviderConfig {
oneof oneof_t2s_cloud_provider_config {
    // Configuration for Eleven Labs text-to-speech provider.
    T2sCloudProviderConfigElevenLabs t2s_cloud_provider_config_elevenlabs = 1;

    // Configuration for Google text-to-speech provider.
    T2sCloudProviderConfigGoogle t2s_cloud_provider_config_google = 2;

    // Configuration for Microsoft text-to-speech provider.
    T2sCloudProviderConfigMicrosoft t2s_cloud_provider_config_microsoft = 3;
}
}

`

Reply:
@teddius You are right. I make it correct and also update the client.